site/app/controllers/calendar/CalendarController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace app\controllers\calendar;
+
+use app\controllers\AbstractController;
+use app\controllers\GlobalController;
+use app\libraries\routers\AccessControl;
+use app\libraries\response\JsonResponse;
+use app\libraries\response\WebResponse;
+use app\libraries\response\MultiResponse;
+use app\models\Course;
+use app\models\gradeable\Gradeable;
+use app\models\gradeable\GradeableList;
+use app\models\User;
+use app\views\calendar\CalendarView;
+use Exception;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CalendarController extends AbstractController {
+    private $gradeables = [];
+
+    /**
+     * Loads from the database of all courses and get all gradeables information. Only load once unless the user
+     * refreshes the page.
+     *
+     * @param null $user_id
+     */
+    private function loadFromDB($user_id = null) {
```

Comment:
Thanks for this suggestion. I'll remove this parameter in the next commit.

Reply:
Only resolve the conversation if you've made the actual commit to the branch and pushed. -GitHub will also show the conversation as outdated once a push has been made to indicate the file has been updated since my requested change is made.

This way we can quickly see if more work needs to be done or not in a PR :)